dirsrvtests/tests/suites/filter/inchain_test.py

```diff
@@ -812,6 +812,27 @@ def test_invalid_assertion(topo):
     memberof = topo.standalone.search_s(DEFAULT_SUFFIX, SCOPE_SUBTREE, "(member:%s:=%s)" % (INCHAIN_OID, user))
     assert len(memberof) == 0
 
+def test_check_dsconf_matchingrule(topo):
+    """Test that the matching rule 'inchain' is listed by dsconf
+
+    :id: b8dd4049-ccec-4316-bc9c-5aa5c5afcfbd
+    :setup: Standalone Instance
+    :steps:
+        1. run 'dsconf instance schema matchingrules list'
+        2. Checks that the output contains inchaineMatch matching rule
+    :expectedresults:
+        1. Success
+        2. Success
+    """
+    dsconf_cmd= ['/usr/sbin/dsconf', topo.standalone.serverid, 'schema', 'matchingrules', 'list']
+    # Set referral
+    log.info("Use dsconf to check defined matching rules")
+    log.info(f'Command used: %{dsconf_cmd}')
+    proc = subprocess.Popen(dsconf_cmd, stdout=subprocess.PIPE)
```

Comment:
Hi Thierry, 
IMHO you should avoid to use subprocess as far as possible. 
In this case you can simply check directly the schema:

schema = Schema(inst)
#mrs = [ str(mr) for mr in schema.get_matchingrules() ]
mrs = [ f"{mr.oid} {mr.names[0]}" for mr in schema.get_matchingrules() if len(mr.names) > 0 ]
print(mrs)
assert  '1.2.840.113556.1.4.1941 inchainMatch' in mrs